opercap.f90

```diff
@@ -0,0 +1,422 @@
+!   Capt'n Oper
+!   Module to house everying specific to captn operator
+!   Neal Avis Kozar 2020
+!   all units of distance: cm
+!   all units of mass/energy : GeV (or GeV/c^2, don't forget)
+!   all units of time: seconds
+!   Sticking with notation of 1504.04378. Cite that paper. Or 1605.06502 it's even better.
+
+
+module opermod
+    use sharedmod
+    implicit none
+    double precision, parameter :: hbar=6.582d-25 !GeV*s
+    !this goes with the Serenelli table format
+    
+    ! integer :: pickIsotope
+    ! double precision :: j_chi
+
+    double precision, parameter :: AtomicNumber_oper(16) = (/ 1., 3., 4., 12., 14., 16., 20., 23., 24., 27., &
+                                                        28., 32., 40., 40., 56., 58./) !the isotopes the catena paper uses
+    character (len=4) :: isotopes(16) = [character(len=4) :: "H","He3","He4","C12","N14","O16","Ne20","Na23","Mg24", &
+                                                                "Al27", "Si28","S32","Ar40","Ca40","Fe56","Ni58"]
+    double precision, parameter :: AtomicSpin_oper(16) = (/ 0.5, 0.5, 0., 0., 1., 0., 0., 1.5, 0., 2.5, &
+                                                        0., 0., 0., 0., 0., 0./) !spins pulled from https://physics.nist.gov/PhysRefData/Handbook/element_name.htm
+    double precision :: coupling_Array(14,2)
+    double precision :: W_array(8,16,2,2,7)
+
+    integer :: q_shared
+    logical :: w_shared
+    
+    contains
+
+    ! having removed the scaling momentum, are the units off here? I'm looking at the p/c0 in particular
+    function GFFI_H_oper(w,vesc,mq)
+        double precision :: p, mu,w,vesc,u,muplus,GFFI_H_oper,G
+        integer mq
+        p = mdm*w
+        mu = mdm/mnuc
+        muplus = (1.+mu)/2.
+        u = sqrt(w**2-vesc**2)
+        if (mq .ne. -1) then
+            G = (p/c0)**(2.d0*mq)*mdm*w**2/(2.d0*mu**mq)*1./(1.+mq)*((mu/muplus**2)**(mq+1)-(u**2/w**2)**(mq+1))
+        else
+            !eps added to make the log finite: the value of eps does not affect the answer
+            G = (p/c0)**(2.d0*mq)*mdm*w**2/(2.d0*mu**mq)*log(mu/muplus**2*w**2/(u)**2)
+        endif
+        GFFI_H_oper = G
+    end function GFFI_H_oper
+    
+    function GFFI_A_oper(w,vesc,A,mq)
+        double precision :: p, mu,w,vesc,u,muplus,mN,A,Ei,B
+        double precision :: dgamic,GFFI_A_oper
+        integer :: mq
+        p = mdm*w
+        mu = mdm/mnuc/A
+        muplus = (1.+mu)/2.
+        u = sqrt(w**2-vesc**2)
+        mN = A*mnuc
+        Ei  = 5.8407d-2/(mN*(0.91*mN**(1./3.)+0.3)**2)
```

Comment:
To be consistent with Catena, should be: 
Ei = 1./(2.*mN*2.671223d2/(45.d0*a**(-1./3.) - 25.d0*a**(-2./3.)))

Reply:
We found that it should be:
`Ei = 1./4.d0/mN/264.114*(45.d0*A**(-1./3.)-25.d0*A**(-2./3.))`